rln-wasm/Makefile.toml

```diff
@@ -0,0 +1,29 @@
+[env]
+CARGO_MAKE_EXTEND_WORKSPACE_MAKEFILE = true
+
+[tasks.pack-build]
+command = "wasm-pack"
+args = ["build", "--release", "--target", "web", "--scope", "waku"]
+
+[tasks.pack-rename]
+script = "sed -i 's/rln-wasm/zerokit-rln-wasm/g' pkg/package.json"
```

Comment:
I'd agree if more changes were required in the `package.json` file, however, since it's only a single string, I prefer to use `sed` since it's available in all unix-like OSs, unlike `jq`, which could require an extra step of installing it.